### ISSUE TITLE
Glue ingestion doesn't work with AWS temporary security credentials

### ISSUE THREAD

**Issue by matef** [open-metadata/OpenMetadata#1522]
I tried to use my temp AWS session credentials to ingest from Glue catalog to local OpenMetadata setup. The Configuration has only two parameters which are the `aws_access_key_id` and `aws_secret_access_key`. There is a missing part here which is the `session_token` to overcome this issue I commented out the part where you populate those values so boto3 could pick up the current session.

```
[2021-12-01 16:36:29,488] ERROR    {metadata.ingestion.source.glue:144} - Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/metadata/ingestion/source/glue.py", line 112, in ingest_tables
    for tables in self.glue.get_tables(DatabaseName=self.config.database)[
  File "/usr/local/lib/python3.9/site-packages/botocore/client.py", line 388, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/usr/local/lib/python3.9/site-packages/botocore/client.py", line 708, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.exceptions.ClientError: An error occurred (UnrecognizedClientException) when calling the GetTables operation: The security token included in the request is invalid.
```

**Comment by harshach** [comment]
@matef working on this, https://github.com/open-metadata/OpenMetadata/issues/1345 .Looks related? if you have any input what's a better way to capture this in config, please let us know. Especially these workflows will run in a scheduler like airflow

**Comment by matef** [comment]
I think the best solution is to make those fields optional and in case they are not supplied skip lines (79,80) and boto3 will pick up the default session. This will make it easy to run ingestion from a local machine as well as from a Lambda function for example. 

https://github.com/open-metadata/OpenMetadata/blob/a9e70e6a89591ac29aaba34664a02150c262d75c/ingestion/src/metadata/ingestion/source/glue.py#L79

**Comment by amiorin** [comment]
@harshach 
You can assign this one to me.

**Comment by amiorin** [comment]
Fixed in #1575
